lldb/source/Plugins/Language/Swift/SwiftLanguage.cpp

```diff
@@ -1730,6 +1735,8 @@ bool SwiftLanguage::GetFunctionDisplayName(
       VariableSP var_sp(args.GetVariableAtIndex(arg_idx));
       ValueObjectSP var_value_sp(
           ValueObjectVariable::Create(exe_scope, var_sp));
+      if (!var_sp || !var_value_sp || var_sp->IsArtificial())
```

Comment:
I assume this is the `self` bit you mentioned in the commit message?

Reply:
Yes.

Reply:
can you explain a little more, where is self being omitted? Is there a test for it that I've overlooked?